app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.includes(:user).where(user: params[:user_id])
+    respond_to do |format|
+      format.html
+      format.json { render json: @posts }
+    end
+  end
+
+  def create
+    post = params[:post]
+    user = current_user
+
+    new_post = Post.new(post.permit(:title, :text))
```

Comment:
- [ ] Let's isolatte permitted params in a private method to decouple the create method from that.
```ruby
private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```

Reply:
- [ ] like the last code reviewer said kindly use another method for params and make it private.